src/aggregate_utils.py

```diff
@@ -1,17 +1,39 @@
-from src.download_utils import get_default_params, download_store_data
+from download_utils import get_default_params, download_store_data
+import re
```

Comment:
I would include the search functions in a separate file, e.g. `src/search_utils.py` due to `import re`.